### DISCUSSION TITLE
Promote Your App

### DISCUSSION THREAD

**Discussion by ratson** [admob-plus/admob-plus#272] [Show and tell]
You are welcome to submit PR as [a plugin user](https://admob-plus.github.io/users).

**Comment by CaturViriyananda** [comment]
i made social web app for Indonesian only. i haven't made enough traffic to generate any money. but i'm very thankful for your work, and i will notify this discussion once my app started to growing later, in the future. i hope.

[here is the App](https://play.google.com/store/apps/details?id=com.ctrvrynnd.tanyacwk)
